packages/flutter/lib/src/material/chip.dart

```diff
@@ -101,83 +113,642 @@ class Chip extends StatelessWidget {
   /// Defaults to a [StadiumBorder].
   final ShapeBorder border;
 
-  /// Color for delete icon, the default being black.
-  ///
-  /// This has no effect when [onDelete] is null since no delete icon will be
-  /// shown.
+  /// Color for delete icon. The default is based on the ambient [IconTheme].
   final Color deleteIconColor;
 
   /// Message to be used for the chip delete button's tooltip.
-  ///
-  /// This has no effect when [onDelete] is null since no delete icon will be
-  /// shown.
```

Comment:
why did you remove these "has no effect" paragraphs?

Reply:
Because Hans asked me to... :-)

Reply:
Hans felt that we were overstating the obvious and compounding the problem with verbiage.